### ISSUE TITLE
Do not wrap RHS of assignment when only a single token

### ISSUE THREAD

**Issue by sjoerdjob** [psf/black#1115]
**Describe the style change**
When the right-hand-side of an assignment is just a single token, it should not be moved to a line of its own.

**Examples in the current _Black_ style**

In a `constants.py` file in our codebase, Black has chosen some formatting which I think is a bit peculiar. Part of it is due to our (in this case) excessively long constant names, which have been anonymized.

```
THIS_IS_AN_EXTREMELY_LONG_NAME_FOR_A_CONSTANT = (
    "THIS_IS_AN_EXTREMELY_LONG_NAME_FOR_A_CONSTANT"
)
SOME_EXTREMELY_LONG_VARIABLE_NAME_THAT_WILL_EVENTUALLY_END_UP_WITH_A_SINGLE_TOKEN_VALUE = (
    5
)
```

Especially the second case is quite confusing, as `5` has the same length as the `(` that's being used to replace it, so this does not even save on maximum line-length.

**Desired style**

I myself would prefer the following formatting

```
THIS_IS_AN_EXTREMELY_LONG_NAME_FOR_A_CONSTANT = "THIS_IS_AN_EXTREMELY_LONG_NAME_FOR_A_CONSTANT"
SOME_EXTREMELY_LONG_VARIABLE_NAME_THAT_WILL_EVENTUALLY_END_UP_WITH_A_SINGLE_TOKEN_VALUE = 5
```

**Comment by erikkjellgren** [comment]
I agree with this one, but also the same consideration for functions with only a single input.
